### PR TITLE
show a useful error when no options are supplied

### LINKED ISSUE #1616: How to use Modernizr v3?
Trying to use Modernizr3 and unfortunately I can't really figure it out. Installing from `npm` does not work:

```js
var modnernizr = require('modernizr')
```

Cloning and installing the repo, then running `./bin/modernizr` leads me to the following error:

```sh
{ [AssertionError: missing path]
  name: 'AssertionError',
  actual: undefined,
  expected: true,
  operator: '==',
  message: 'missing path',
  generatedMessage: false }
```

### PR DESCRIPTION
**Pull request by patrickkettner**
fixes #1616

### CODE REVIEW

File: bin/modernizr [hunk 1]
```diff
@@ -150,10 +150,14 @@ if (argv.c) {
 }
 
 try {
-  if (!inlineConfig) {
+  if (!inlineConfig && configPath) {
     config = require(configPath);
   } else if (configPath) {
     config = _.extend(config, require(configPath));
+  } else if (!configPath) {
+    console.error('config flie, inline features, or options required.');
```
Comment (ryanseddon) [inline bin/modernizr]: `flie` -> `file`

Comment (patrickkettner) [inline bin/modernizr]: :[

**Comment by ryanseddon** [comment]
Looks like some of the packages failed in appveyor. Maybe just needs to be ran again.

**Comment by ryanseddon** [comment]
:+1:

**Pull request merged**